src/Core/Plugin.php

```diff
@@ -170,6 +184,24 @@ public static function load($plugin, array $config = [])
     }
 
     /**
+     * Loads the config mappings for plugins.
+     *
+     * The map data in CONFIG/plugins.php map data is used set default paths
+     * for where plugins are located.
+     *
+     * @return void
+     */
+    protected static function _loadConfigMap()
+    {
+        if (!file_exists(CONFIG . 'plugins.php')) {
+            static::$_config = [];
+            return;
+        }
+        $config = include CONFIG . 'plugins.php';
```

Comment:
@AD7six @josegonzalez Does composer have a way to query which packages arr installed and what their types are?